pkgs/development/python-modules/websockets_client/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi,
+  six, backports_ssl_match_hostname, unittest2, argparse }:
+buildPythonPackage rec {
+  pname = "websocket_client";
+  version = "0.46.0";
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "0f1p9zkzpw0q1cqwnfn788mj3fjqg5z3vsd9pjnz50dk12znngwk";
+  };
+
+  checkInputs = [ unittest2 ];
+  propagatedBuildInputs = [ six backports_ssl_match_hostname argparse ];
```

Comment:
All but `six` aren't required for the supported Python versions: https://github.com/websocket-client/websocket-client/blob/master/setup.py

Reply:
I have dropped all the dependencies that are not used for the python versions we have. Only six and unittest2 remains.

Thanks again.

Reply:
Why does `unittest2` remain?

Reply:
because I forgot to drop it… done